src/main/groovy/SDKExample.groovy

```diff
@@ -0,0 +1,98 @@
+import com.profitbricks.sdk.model.*
+import static com.profitbricks.sdk.Commands.*
+
+class SDKExample {
```

Comment:
The example is intended to show the users a certain flow of operations, i.e. creating a data center and adding a server to it, attaching volumes to the server and adding a lan.

The `SDKTester` tests all operations for the purpose of making sure that each operation works.